it/connectors/DeleteOrIgnoreExpensesConnectorSpec.scala

```diff
@@ -18,10 +18,10 @@ package connectors
 
 import com.github.tomakehurst.wiremock.http.HttpHeader
 import connectors.httpParsers.CreateOrAmendExpensesHttpParser.CreateOrAmendExpensesResponse
-import models.expenses.CreateExpensesRequestModel
+import models.expenses.createUpdate
+import models.expenses.createUpdate.CreateUpdateExpensesRequest
```

Comment:
again do we need both. I think we can just use one ?

Reply:
I can’t see this. Maybe it was already removed by my rebase as there is only one?